core/src/main/java/io/aiven/klaw/repository/OperationalRequestsRepo.java

```diff
@@ -0,0 +1,23 @@
+package io.aiven.klaw.repository;
+
+import io.aiven.klaw.dao.OperationalRequest;
+import io.aiven.klaw.dao.OperationalRequestID;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.data.repository.query.QueryByExampleExecutor;
+
+public interface OperationalRequestsRepo
+    extends CrudRepository<OperationalRequest, OperationalRequestID>,
+        QueryByExampleExecutor<OperationalRequest> {
+  Optional<OperationalRequest> findById(OperationalRequestID operationalRequestID);
+
+  List<OperationalRequest> findAllByTenantId(int tenantId);
+
+  @Query(
+      value = "select max(reqid) from kwoperationalrequests where tenantid = :tenantId",
+      nativeQuery = true)
+  Integer getNextOperationalRequestId(@Param("tenantId") Integer tenantId);
```

Comment:
why don't we use database sequences to get next id?
This code could lead to issue if there are more than 1 app connected to the same database

Reply:
I think that should not happen. For now klaw is designed for it's backend only to connect to its database.

However we can come up with a proposal to use sequences

Reply:
yes, however if we are talking about HA then there are could be several nodes connected to the same database...
Or did I miss anything?

anyway yes, it seems a separate task